inventory/group_vars/hu.yml

```
@@ -0,0 +1,23 @@
+---
+
+# See https://github.com/openfoodfoundation/ofn-install/wiki/Setup for more info
+checkout_zone: Europe
+country_code: HU
+default_country_code: HU
+currency: HUF
+locale: hu_HU                  # Default locale for OFN instance
+available_locales: hu    # Available locales for OFN instance
+language: hu_HU.UTF-8       # Default language for the server
+language_packages:          # Language packages for the server
+  - language-pack-hu-base
+l10n_repo: https://github.com/pragai/I18n_hu.git
+timezone: Europe/Budapest
+
+
+# Used for the LetsEncrypt email setting
+developer_email: pragai.robert@abasys.hu
+
+# Users listed here will have their key from /files/keys added to the app_user
+users_sysadmin:
+  - pragai
```

If you wish the global admin team to have access to your server so that we can manage it:
```suggestion
  - "{{ core_devs }}"
  - pragai
```

---

Added core_devs here.